imagersite/templates/imagersite/home.html

```diff
@@ -1,15 +1,20 @@
 {% extends "imagersite/base.html" %}
 {% block body %}
+
     <h1>Imagersite</h1>
-    <img src="media/home.jpeg" width="500">
+    {% if photo.file %}
+      <img src="media/{{ photo.file }}" width="500">
+    {% else %}
+      <img src="media/home.jpeg" width="500">
+    {% endif %}
+
     <p>Join the site or I will kill you!</p>
```

Comment:
If I'm logged in, I shouldn't see the next 5 lines. It wouldn't make much sense to prompt me to log in again.